### PR TITLE
Typescript test

### PR DESCRIPTION
**Pull request by sean-r-martin**

### CODE REVIEW

File: ng2ts/tests/codelabTest.ts [hunk 1]
```diff
@@ -56,6 +56,11 @@ describe('Component', () => {
      * e.g. if the user created teh class, but haven't exported it this
      * test will still pass.
      */
+
+    const codeString = typescript_intro_Codelab_ts_AST.text;
+    const regEx = RegExp(/(Codelab) *\(\)/g);
+    const noParentheses = codeString.search(regEx) === -1;
+    chai.expect(noParentheses).equals(true);
```
Comment (prestonvanloon) [inline ng2ts/tests/codelabTest.ts]: Not familiar with chai, but can't you do `expect(codeString).not.toMatch(/(Codelab) *\(\)/g)` like jasmine?

Comment (tessaSAC) [inline ng2ts/tests/codelabTest.ts]: there is definitely a `to.not.equal` option (and `to.not.eql`)

Comment (kirjs) [inline ng2ts/tests/codelabTest.ts]: there's http://chaijs.com/api/bdd/ match

**Comment by kirjs** [comment]
This is fixed in #428

**Pull request closed without merging**